coordinator/core/storewrapper.go

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) Edgeless Systems GmbH.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package core
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"encoding/json"
+	"strconv"
+	"strings"
+
+	"github.com/edgelesssys/marblerun/coordinator/manifest"
+)
+
+const (
+	requestActivations string = "activations"
+	requestCert        string = "certificate"
+	requestManifest    string = "manifest"
+	requestPrivKey     string = "privateKey"
+	requestSecret      string = "secret"
+	requestState       string = "state"
+	requestUser        string = "user"
+)
+
+// Store is the interface for state transactions and persistance
+type Store interface {
```

Comment:
Should `Store` be in a separate module?
Not sure what's best practice here @thomasten?
Another thought: Should `Get` and `Put` have the prefix/family as an argument instead of joining them in the wrapper?
This would leave it to the plugins to handle the prefix/family. Some storage backend might support this for performance reasons.

Reply:
I think it is better to have `Get` and `Put` only take a single argument as the key. This leaves the interface as a simple key-value store.
I dont think the performance impact of joining or splitting a short string will be noticeable, even with thousands of requests per second.

Reply:
I agree with @m1ghtym0 regarding separate module.
@daniel-weisse It's not about string operation cost, but KV stores that support grouping keys. However, I think this is negligible for our use case. I don't have a strong opinion, only a slight preference to keep it simple.